ddl.dic

```diff
@@ -76,17 +76,21 @@ save_alias.deprecation_date
 
     _definition.id                '_alias.deprecation_date'
     _definition.class             Attribute
-    _definition.update            2013-09-08
+    _definition.update            2024-02-05
     _description.text
 ;
     Date that the aliased tag was deprecated as a definition tag.
+
+    An unquoted value of '.' or '?' indicates that the associated alias has not
```

Comment:
Mentioning `?` explicitly is unnecessary, I think, as the behaviour of `?` and missing attributes will always be to adopt the default value, no need to reiterate that here.

Reply:
Ok, I removed the mention of `?`.